packages/gatsby-plugin-styletron/src/gatsby-browser.js

```diff
@@ -1,8 +1,16 @@
 const React = require(`react`)
 const styletron = require(`./index`)
-const { Provider } = require(`styletron-react`)
+const { Provider, DebugEngine } = require(`styletron-react`)
+
+const debug = process.env.NODE_ENV === `production` ? void 0 : new DebugEngine()
```

Comment:
Could/should this also be configurable? e.g. with pluginOptions?

Reply:
Yep! You can now disable it:

```js
module.exports = {
  plugins: [
    {
      resolve: `gatsby-plugin-styletron`,
      options: {
        debug: false
      },
    },
  ],
}
```

**Dev ENV:** By default, it's enabled. Can be disabled through `options`.
**Prod ENV:** Always disabled. It adds quite an overhead, so it should never be used in production.